ptx/sec_deriv_intro.ptx

```diff
@@ -1493,11 +1502,14 @@
           <caption>A graph of <m>y=x^{1/2}</m> and <m>y=x^{3/2}</m> in <xref ref="ex_diff_closed1">Example</xref></caption>
           <image xml:id="img_diff_closed1" width="47%">
             <shortdescription>
-              Two curves one of function x squared and the other of x to the 3/2.  
+              Two curves one of function of square root of x and the other of x to the power 3/2.  
```

Comment:
Two curves are shown: the graph of the square root of x, and the graph of x to the power 3/2.